js/exercise-02.js

```diff
@@ -0,0 +1,19 @@
+/* 
+    1. Declare a variable using let with any name you like, but without a value.
+    2. On a separate line, assign the variable the String: "Moo!"
+    3. Log out this variable to the terminal using console.log
+    
+    Repeat the above steps but try using a const variable. Comment this new part of code and explain what is happening.
+*/
+
+let vaca;
+vaca = "Moo!";
+console.log(vaca);
+
+const vaca; 
+vaca = "Moo!";
+console.log(vaca);
+
+/* const es una variable "inmutable" per tant no pots reassignar el valor de la variable 
```

Comment:
⚠️ Be careful! 

When we define a variable as `const` it does not mean it is `immutable`. This is a quite complex concept in JS and we will explain it.

A `const` variable cannot be reassigned once it has been initialised, that's it.

FIX: review explanation